modules/element-manager/assets/js/app-editor-one/components/PromotionWidgetsTable.js

```diff
@@ -0,0 +1,139 @@
+import {
+	Stack,
+	Box,
+	Tooltip,
+	Switch,
+	IconButton,
+	Typography,
+	Table,
+	TableHead,
+	TableBody,
+	TableRow,
+	TableCell,
+	TableContainer,
+	Paper,
+} from '@elementor/ui';
+import { HelpIcon } from '@elementor/icons';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+import { UpgradeButton } from '../../upgrade-button';
+import { EditButtonDisabled } from './RolePermissions';
+
+export const PromotionWidgetsTable = ( { widgets, promotionData } ) => {
+	const { element_manager: elementManager } = promotionData;
+
+	if ( ! widgets.length ) {
+		return null;
+	}
+
+	return (
+		<>
+			<Box sx={ { mt: 5, mb: 2.5 } }>
+				<Stack
+					direction="row"
+					justifyContent="space-between"
+					alignItems="center"
+				>
+					<Box>
+						<Typography variant="h6" component="h3">
+							{ __( 'Elementor Pro Elements', 'elementor' ) }
+						</Typography>
+						<Typography variant="body2" component="p">
+							{ __( 'Unleash the full power of Elementor\'s features and web creation tools.', 'elementor' ) }
+						</Typography>
+					</Box>
+					<Box>
+						<UpgradeButton
+							href={ elementManager.url }
+							text={ elementManager.text }
+							className="e-id-elementor-element-manager-button-upgrade-pro-elements"
+						/>
+					</Box>
+				</Stack>
+			</Box>
+			<TableContainer component={ Paper } variant="outlined">
+				<Table size="small">
+					<TableHead>
+						<TableRow>
+							<TableCell sx={ { width: '200px' } }>
```

Comment:
In Elementor UI we usually use sx unit styling like. Can you please check if this is possible?

```javascript
sx={ { width: 25 } }
```